DiamondLightSource/hyperion
# 1350 nexus file cannot be read by imginfo

Fixes #1350 

Link to nexgen PR (if required): DiamondLightSource/nexgen#237
(remember to update `setup.cfg` with the dodal commit tag if you need it for tests to pass!)

### To test:
1. Imginfo and autoPROC pipeline run successfully against generated nexus files

* The nexgen write operation has been moved even though now I believe it's not necessary since there's no update method in nexgen anymore, but I have left in rather than move it back, I am guessing that perhaps it may come in handy in future if we ever need to do it? If people would rather I restore the old ordering I can do this
* Running it as system test requires DOCKER environment variable to be set to `podman` if you want to use podman otherwise it defaults to `docker`. Same goes for the utility scripts
* System tests is marked with `s03` but not sure if we want a different flag
* System tests require you to have run `build_imginfo.sh` to build `imginfo`
* `imginfo` is pinned to the latest commit hash, do we want to track the `HEAD` - they don't appear to have version tags
* I'm assuming the reference imginfo output is correct for these files.
